dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -3530,18 +3501,73 @@ def test_options_ingested_correctly(self):
         self.assertFalse(self.profiler2.options.data_labeler.is_enabled)
 
 
-class TestStructuredProfilerNullValues(unittest.TestCase):
+class TestStructuredProfilerRowStatistics(unittest.TestCase):
     def setUp(self):
         test_utils.set_seed(0)
 
+    @classmethod
+    def setUpClass(cls):
+        test_utils.set_seed(seed=0)
+
+        data = {
```

Comment:
none of the null tests use this, correct?

Future, I would suggest setting up a class dataset to be usable by all tests. So it would handle cases we want to test for unique and null. What I would see missing here would be the all null cases etc.

not to update in this, but if we want to throw up a good first issue that describes a refactor for that purpose here, that'd be ideal.